app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java

```diff
@@ -253,8 +296,8 @@ private void fixup(final int changed_row)
         // Assert one empty row at bottom
         final int len  = data.size();
         if (len <= 0  ||
-            (data.get(len-1).getName().trim().length() > 0  &&
-             data.get(len-1).getValue().trim().length() > 0) )
+                (data.get(len-1).getName().trim().length() > 0  &&
+                        data.get(len-1).getValue().trim().length() > 0) )
             data.add(new MacroItem("", ""));
```

Comment:
I was actually proud of this formatting, how the data.get(.. lined up for the getName and getValue, but whatever...

Reply:
Sorry, as you can see I have set the brackets to follow your format. I shall fix this.

Reply:
These lines are a good example for auto-format simply not understanding what belongs together.

As for all the rest, the whole "<enter name>" stuff could also be removed, but now that you already put in the time to clear it, go ahead.

Reply:
For many, directly clicking on the table is easier than the button... do you see any errors in the code

Reply:
Hah, web page, can't just enter 'less than'. ...
Meant to say the whole "enter name" stuff .. could be removed, would be less code overall.
But again, go ahead, I've also often ended up with that 'hint' text in the name or value.

Reply:
thanks